src/ArduinoCellular.cpp

```diff
@@ -214,12 +217,17 @@ bool ArduinoCellular::awaitNetworkRegistration(){
     if(this->debugStream != nullptr){
         this->debugStream->println("Waiting for network registration...");
     }
-    while (!modem.waitForNetwork()) {
+    while (!modem.waitForNetwork(20000L)) {
         if(this->debugStream != nullptr){
             this->debugStream->print(".");
         }
+#if defined(ARDUINO_ARCH_MBED)
+        if(mbed::Watchdog::get_instance().is_running()) {
+            mbed::Watchdog::get_instance().kick();
+        }
+#endif
```

Comment:
Please indent this block

Reply:
4 instead of 8 spaces :-)

```
    #if defined(ARDUINO_ARCH_MBED)
        if(mbed::Watchdog::get_instance().is_running()) {
            mbed::Watchdog::get_instance().kick();
        }
    #endif
```

Reply:
The snippet is inside the while loop so 8 spaces shoud be correct